app/Models/ReservationGroup.php

```
@@ -0,0 +1,299 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+use Carbon\Carbon;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+use App\Models\Reservation;
+
+class ConflictException extends \Exception
+{
+}
+
+class ReservationGroup extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'default_item',
+        'user_id',
+        'frequency',
+        'default_title',
+        'default_from',
+        'default_until',
+        'default_note',
```

I find the terminology default misleading. What about 'group_' as prefix?